### ISSUE TITLE
Sort by size does not work

### ISSUE THREAD

**Issue by NEOTRONICS** [Raylehnhoff/SCFSD#84]
Select different ships sorted by size, the sorting is not according to sizes, at least not always ( Used Browser: Opera 49 ).

**Comment by NEOTRONICS** [comment]
My mistake, had the function expected differently.

**Comment by Raylehnhoff** [comment]
@NEOTRONICS -- what was the expected behavior?

**Comment by NEOTRONICS** [comment]
I thought the ship list ( generated picture ) can be rearranged with this option, but it's just for the definition list. As written my fault, sorry :) .

**Comment by Raylehnhoff** [comment]
It re-arranges the fleet graphic AND the input list, but does so based on the size of the SVG and not necessarily the actual ships dimensions (though they should be _pretty close_)

Once it re-arranges the fleet graphic, you can generate a new png output.

I don't use Opera at all, and the generator is broken (in Firefox) due to a VERY longstanding bug in Firefox and the Canvas plugin I use. 

If you're not seeing it re-organize the ship list, maybe give it a whirl in Chrome?

**Comment by NEOTRONICS** [comment]
No, that's not how it works for me ... fantastic ... browser support is a crazy thing :D. 

If I have sorted by name and select the ships in the correct order everything is fine.  So often i do not change the fleet, only in each concept Sale - almost X). Then a hint might not be bad for which browser this page is optimized/tested and good.

**Comment by Raylehnhoff** [comment]
Let's reopen this. I'll pull opera down this evening and see if I can fix
it.

**Comment by Raylehnhoff** [comment]
I downloaded Opera on Windows 10 with no plugins, and the sorting system is working as I'd expect it to, re-organizing both the input list and the ship list.

Can you please open the Opera dev tools (right click a page element->inspect element), click the console tab.

Refresh the page and re-sort a few times. In the developer tools window, if there are any console errors, please paste them in this thread. 

Also, in the console, please type:

`JSON.stringify(localStorage);`

and hit enter. Please paste the output to this thread. This will let me test with the same ship inputs that you're using for the site and tell me if you encountered any errors along the way.

Also, what operating system are you using?
